### PR TITLE
fix(clouddriver-aws) Unable to pull ec2 instance list in AWS China region

### PR DESCRIPTION
**Pull request by wald-e**
Cannot pull ec2 instance list in AWS China region

https://github.com/spinnaker/spinnaker/issues/6637

### CODE REVIEW

File: clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java [hunk 1]
```diff
@@ -86,13 +86,17 @@ public AmazonInstanceTypeCachingAgent(
...
       pricingHost = HttpHost.create("https://pricing.us-east-1.amazonaws.com");
+      pricingUri = URI.create("/offers/v1.0/aws/AmazonEC2/current/" + region + "/index.json");
+      this.httpClient = httpClient;
+
     }
-    pricingUri = URI.create("/offers/v1.0/aws/AmazonEC2/current/" + region + "/index.json");
-    this.httpClient = httpClient;
   }
 
+
```
Comment (mattgogerly) [inline clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java:99]: Newline not needed

File: clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java [hunk 2]
```diff
@@ -86,13 +86,17 @@ public AmazonInstanceTypeCachingAgent(
     this.credentialsRepository = credentialsRepository;
     if (region.startsWith("cn-")) {
       pricingHost = HttpHost.create("https://pricing.cn-north-1.amazonaws.com.cn");
+      pricingUri = URI.create("/offers/v1.0/aws/AmazonEC2/current/index.json");
+      this.httpClient = httpClient;
     } else {
       pricingHost = HttpHost.create("https://pricing.us-east-1.amazonaws.com");
+      pricingUri = URI.create("/offers/v1.0/aws/AmazonEC2/current/" + region + "/index.json");
```
Comment (mattgogerly) [inline clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java:93]:
```suggestion
      pricingUri = URI.create(String.format("/offers/v1.0/aws/AmazonEC2/current/%s/index.json", region));
```

File: clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java (hunk 1 above)
Comment (wald-e) [reply clouddriver-aws/src/main/java/com/netflix/spinnaker/clouddriver/aws/provider/agent/AmazonInstanceTypeCachingAgent.java]: deleted

**Comment by mattgogerly** [comment]
Please run spotless:apply to fix the build

**Comment by wald-e** [comment]
Will file a new PR

**Pull request closed without merging**